tests/e2e.test.js

```diff
@@ -49,49 +49,53 @@ beforeAll(() => {
 describe.each([
   ['desktop', DEFAULT_DEVICE],
   ['mobile', devices['Nexus 7']],
-])('General (%s)', (name, device) => {
-  beforeEach(async () => {
-    await page.emulate(device);
-    await page.goto(url.href);
-  });
+])(
+  'General (%s)',
+  (name, device) => {
+    beforeEach(async () => {
+      await page.emulate(device);
+      await page.goto(url.href);
+    });
 
-  it('produces a screenshot', async () => {
-    await page.screenshot({
-      path: path.resolve(ARTIFACTS_DIR, `${name}.png`),
+    it('produces a screenshot', async () => {
+      await page.screenshot({
+        path: path.resolve(ARTIFACTS_DIR, `${name}.png`),
+      });
     });
-  });
 
-  it('displays the text "Simple Icons"', async () => {
-    await expect(page).toMatch('Simple Icons');
-  });
+    it('displays the text "Simple Icons"', async () => {
+      await expect(page).toMatch('Simple Icons');
+    });
 
-  it('has the search input in view on load', async () => {
-    const $searchInput = await page.$('#search-input');
-    expect(await isInViewport($searchInput)).toBeTruthy();
-  });
+    it('has the search input in view on load', async () => {
+      const $searchInput = await page.$('#search-input');
+      expect(await isInViewport($searchInput)).toBeTruthy();
+    });
 
-  it('has the order controls in view on load', async () => {
-    const $orderAlphabetically = await page.$('#order-alpha');
-    expect(await isInViewport($orderAlphabetically)).toBeTruthy();
+    it('has the order controls in view on load', async () => {
+      const $orderAlphabetically = await page.$('#order-alpha');
+      expect(await isInViewport($orderAlphabetically)).toBeTruthy();
 
-    const $orderByColor = await page.$('#order-color');
-    expect(await isInViewport($orderByColor)).toBeTruthy();
-  });
+      const $orderByColor = await page.$('#order-color');
+      expect(await isInViewport($orderByColor)).toBeTruthy();
+    });
```

Comment:
Why all of these changes? Please, don't touch code that doesn't correspond to this PR.

Reply:
the PR is taking too long, that means lot of merges, which means that code change in places that are not directly related to it